src/en/troubleshooting-logs-audit.md

```diff
@@ -0,0 +1,28 @@
+Title: Audit logging
+
+# Audit logging
```

Comment:
I don't understand why this has been made a tiny page on its own and not part of the logging page (which it even tells you to read)